src/main/java/net/minecraftforge/client/event/GuiScreenEvent.java

```diff
@@ -128,6 +128,58 @@ public Post(GuiScreen gui, int mouseX, int mouseY, float renderPartialTicks)
         }
     }
     
+    public static class DrawHoverTextEvent extends GuiScreenEvent
+    {
+        /**
+         * The x coordinate of the mouse pointer on the screen.
+         */
+        public final int mouseX;
+        /**
+         * The y coordinate of the mouse pointer on the screen.
+         */
+        public final int mouseY;
+        /**
+         * The text to be displayed.
+         */
+        public final List textLines;
+        
+        public DrawHoverTextEvent(GuiScreen gui, int mouseX, int mouseY, List textLines)
+        {
+            super(gui);
+            this.mouseX = mouseX;
+            this.mouseY = mouseY;
+            this.textLines = textLines;
+        }
+        
+        /**
+         * This event fires just before {@code GuiScreen.drawHoveringText()} is called.
+         * Cancel this event to skip {@code GuiScreen.drawHoveringText()}.
+         * 
+         * @author iLexiconn
+         */
```

Comment:
Fixed in my new pr (#2212)
Op 27 nov. 2015 01:16 schreef "LexManos" <notifications@github.com>:

> In src/main/java/net/minecraftforge/client/event/GuiScreenEvent.java
> <https://github.com/MinecraftForge/MinecraftForge/pull/2041#discussion_r46011082>
> :
>
> > +        public final List textLines;
> > +
> > +        public DrawHoverTextEvent(GuiScreen gui, int mouseX, int mouseY, List textLines)
> > +        {
> > +            super(gui);
> > +            this.mouseX = mouseX;
> > +            this.mouseY = mouseY;
> > +            this.textLines = textLines;
> > +        }
> > +
> > +        /**
> > +         * This event fires just before {@code GuiScreen.drawHoveringText()} is called.
> > +         * Cancel this event to skip {@code GuiScreen.drawHoveringText()}.
> > +         *
> > +         * @author iLexiconn
> > +         */
>
> no
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/MinecraftForge/MinecraftForge/pull/2041/files#r46011082>
> .
>